src/Castle.Windsor/MicroKernel/SubSystems/Configuration/DefaultConfigurationStore.cs

```diff
@@ -181,7 +181,8 @@ public IConfiguration[] GetInstallers()
 
 		public IResource GetResource(String resourceUri, IResource resource)
 		{
-			if (resourceUri.IndexOf(Uri.SchemeDelimiter) == -1)
+			var uriSchemeDelimiter = "://";
```

Comment:
> Uri.SchemeDelimiter comes back in 2.0. @jonorossi at you ok with adding something like FEATURE_URIMETHODS for this?

Hmm, GitHub didn't tell me you commented after I loaded the page. I guess I'm looking in the wrong place for .NET Standard 2.0 APIs.

No idea how long we'll support .NET Standard 1.x for. Go for it, would be good to get `Uri.SchemeDelimiter` explicitly mentioned in Windsor's list of conditional symbols like Castle Core.